README.md

```diff
@@ -1,3 +1,35 @@
 ![Build Status](https://travis-ci.org/adtechpotok/go-common.svg?branch=master)
 
 ## Общие либы гошных демонов
+
+##DB writer 
+It sends data to database. If data base is unavailable, after 2 tries it will write into files in configurated directory. When connection is back, it will resend data to db in same query orders.
```

Comment:
Grammarly предложил так поправить:

It sends data to a database. If the database is unavailable, after 2 tries it will write into files in the configurated directory. When the connection is back, it will resend data to DB in same query orders.
Data table must contain field_server_id, to support max instanced daemons